index.html

```diff
@@ -34,16 +27,14 @@ <h1 class="header">I’m Ngala Mac
                 <div class="links">
                     <h3><a href="#">LET’S CONNECT</a></h3>
                     <ul>
-                        <li><img src="./Images/twitter.png"></li>
-                        <li><img src="./Images/Linkedin icon.png"></li>
```

Comment:
- [ ] It will be better 😃 if you put the content of this section at the top little as you see in your design the content is consistent at the bottom of the section, so let's 💯 make the design like the Figma Design by moving the content of the section at top little to stick to the Figma and make the design nice for the users 😊💯
- [ ] Also consider increasing the height of the section a little to stick to the Figma design and to fix the position of the content in this section 🚀💯


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
| ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/97158608-b493-4835-9ca2-c999c4a22e03)  |    ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/e87b1671-a509-495f-ac00-9d0b314662af) |